maven-plugin/src/main/java/io/confluent/kafka/schemaregistry/maven/derive/schema/avro/DeriveAvroSchemaPrimitive.java

```diff
@@ -36,23 +36,23 @@ public class DeriveAvroSchemaPrimitive {
   static {
 
     // Mapping class to data type for avro
-    classToDataTypeAvro.put("class com.fasterxml.jackson.databind.node.DoubleNode", "double");
-    classToDataTypeAvro.put("class com.fasterxml.jackson.databind.node.TextNode", "string");
-    classToDataTypeAvro.put("class com.fasterxml.jackson.databind.node.BigIntegerNode", "double");
-    classToDataTypeAvro.put("class com.fasterxml.jackson.databind.node.IntNode", "int");
-    classToDataTypeAvro.put("class com.fasterxml.jackson.databind.node.LongNode", "long");
-    classToDataTypeAvro.put("class com.fasterxml.jackson.databind.node.BooleanNode", "boolean");
-    classToDataTypeAvro.put("class com.fasterxml.jackson.databind.node.NullNode", "null");
+    classToDataTypeAvro.put("DoubleNode", "double");
```

Comment:
It's best to avoid hard-coded strings, so `"DoubleNode"` should be replaced with `com.fasterxml.jackson.databind.node.DoubleNode.class.getName()`.  (Also use `getName()` instead of `getSimpleName()` to avoid ambiguities).

Reply:
I misunderstood it the last time. I have added it as per your suggestion.